week1/preparation.md

```diff
@@ -0,0 +1,25 @@
+# Preparation
+
+What is version control: https://www.atlassian.com/git/tutorials/what-is-version-control (5 min)
+
+What is git: https://www.atlassian.com/git/tutorials/what-is-git (6 min)
+
+And we'll be roughly following the materials here: https://github.com/HackYourFuture-CPH/Git (Lectures 1,2,3)
```

Comment:
change link to https://github.com/HackYourFuture-CPH/Git/blob/git-improvements/week1/review.md ?